src/formats/PDB.cpp

```diff
@@ -813,21 +804,13 @@ void PDBFormat::write_next(const Frame& frame) {
         if (is_atom_record[bond[0]] && is_atom_record[bond[1]]) { // both must be standard residue atoms
             continue;
         }
-        if (bond[0] > 99999 || bond[1] > 99999) {
+        if (bond[0] > 87440031 || bond[1] > 87440031) {
```

Comment:
Instead of a magic number, how about making this a `const` or `constexpr` value? Like `constexpr size_t MAX_HYBRID36_NUMBER = 87440031;`?